code/game/objects/items/weapons/stunbaton.dm

```diff
@@ -61,6 +71,8 @@
 
 /obj/item/melee/baton/examine(mob/user)
 	..(user)
+	if(isrobot(loc))
+		return
```

Comment:
Whoops! I meant to put a unique examine text in there since robots don't really have a charge percent for the baton as such. Must've missed that during testing.

I'll change that in a bit.